### PR TITLE
feat: Bump cert-manager to 1.9.1

### PR DESCRIPTION
**Pull request by mesosphere-teamcity**
This upgrades cert-manager to 1.9.1 to use the latest upstream chart cert-manager v1.9.1.

Depends on https://github.com/mesosphere/charts/pull/1357

### CODE REVIEW

**Comment by gracedo** [comment]
Konvoy recently bumped cert-manager to 1.9.1 so we can pull this in!

File: services/cert-manager/1.9.1/root-ca/root-ca.yaml [hunk 1]
```diff
@@ -1,6 +1,5 @@
 # a quick solution to fix kommander-traefik and kube-oidc-proxy certificate in attached clusters (with and without cert-manager pre-installed)
-# https://jira.d2iq.com/browse/D2IQ-84510
----
+# https://jira.d2iq.com/browse/D2IQ-84510---
```
Comment (takirala) [inline services/cert-manager/1.9.1/root-ca/root-ca.yaml:2]: @bcmendoza who/what is optimizing these? can we make our automation little bit less smart :see_no_evil: 
nbd and prolly this happens only once but just curious

**Comment by takirala** [comment]
Testing this at https://github.com/mesosphere/kommander/pull/2363

**Comment by gracedo** [comment]
https://github.com/mesosphere/kommander/pull/2363 is ✅

**Review by gracedo — APPROVED** [review]

File: services/cert-manager/1.9.1/root-ca/root-ca.yaml (hunk 1 above)
Comment (cwyl02) [reply services/cert-manager/1.9.1/root-ca/root-ca.yaml:2]: I think we still need to manually fix this for now

**Review by cwyl02 — CHANGES_REQUESTED** [review]

**Review by cwyl02 — APPROVED** [review]

**Pull request merged**